idp_data/idp_data/admin.py

```diff
@@ -1,3 +1,8 @@
 from django.contrib import admin
+from .models import Event, Category, Municipality, EventAction
 
-# Register your models here.
+
+admin.site.register(Event)
+admin.site.register(Category)
+admin.site.register(Municipality)
+admin.site.register(EventAction)
```

Comment:
we don't need this now that we have it inline

Reply:
removed the unnecessary ones